src/php/Printer/Printer.php

```diff
@@ -145,7 +145,7 @@ private function creatScalarTypePrinter(mixed $form): TypePrinterInterface
             return new NullPrinter($this->withColor);
         }
 
-        if ($printerName === 'array' && !$this->readable) {
```

Comment:
Any idea about the reason for adding this `!$this->readable` in the first place?

Reply:
Honestly, I don't remember. Maybe Jens?

Reply:
Honestly, I don't remember. Maybe @jenshaase?

Reply:
@JesusValera @Chemaclass We use the Printer for two purposes:

1. For the Phel function `(print-str ...)` to print a value
2. In the compiler to emit valid Phel code (`OutputEmitter`)

For the second case we use the `readable` flag to only output valid Phel code. In this case a PHP array cannot be converted to Phel code and therefore is ignored. But for the `(print-str ...)` we want to print the array to show it to the user.

Reply:
@jenshaase as follow up question, is this PR-changes helping (removing the `&& !$this->readable`) or should we fix the original issue somewhere else and keep that flag check here? 
> In such a case, where should we look at? 👀

Reply:
@jenshaase as follow up question, is this PR-changes helping (removing the `&& !$this->readable`) or should we fix the original issue somewhere else and keep that flag check here?
In such a case, where should we look at? 👀

Reply:
The error is probably somewhere else. Looks like the build wants to emit some PHP array, which it shouldn't. You probably have then code that is not executable.

Reply:
You are right, @jenshaase. I think the issue is coming from the web_skeleton, after building the it generates the `app.php` and in the second build then it `require_once $targetFile;` and that's what's creating that output
![Screenshot 2024-04-16 at 18 15 57](https://github.com/phel-lang/phel-lang/assets/5256287/194839e7-fdd5-430f-9887-948106b347c4)

![Screenshot 2024-04-16 at 18 14 24](https://github.com/phel-lang/phel-lang/assets/5256287/474475f8-6cdd-43f6-b795-b5532527f83d)

I think we could add a configuration list to set "NoCacheWhenBuilding" files/paths form the `phel-config.php`. This way, the user is responsible to disable the cache for the entry point of the apps, and this way fix this issue. That's my assumption so far.